bot/cogs/color.py

```diff
@@ -257,6 +256,13 @@ async def edit_hex_color(
                 ephemeral=True,
             )
 
+        if re.match(r'^\d+$', role):
```

Comment:
[isdigit()](https://docs.python.org/3/library/stdtypes.html#str.isdigit) is more simple